src/Compilers/CSharp/Portable/PublicAPI.Shipped.txt

```diff
@@ -3266,8 +3263,6 @@ override Microsoft.CodeAnalysis.CSharp.Syntax.XmlTextSyntax.Accept(Microsoft.Cod
 override Microsoft.CodeAnalysis.CSharp.Syntax.XmlTextSyntax.Accept<TResult>(Microsoft.CodeAnalysis.CSharp.CSharpSyntaxVisitor<TResult> visitor) -> TResult
 override Microsoft.CodeAnalysis.CSharp.Syntax.YieldStatementSyntax.Accept(Microsoft.CodeAnalysis.CSharp.CSharpSyntaxVisitor visitor) -> void
 override Microsoft.CodeAnalysis.CSharp.Syntax.YieldStatementSyntax.Accept<TResult>(Microsoft.CodeAnalysis.CSharp.CSharpSyntaxVisitor<TResult> visitor) -> TResult
-override sealed Microsoft.CodeAnalysis.CSharp.CSharpSyntaxNode.ToFullString() -> string
-override sealed Microsoft.CodeAnalysis.CSharp.CSharpSyntaxNode.ToString() -> string
```

Comment:
No.  Some of the overrides were removed because the core functionality was moved up to the base class.  The calls are still virtual, it's just not necessayr for VB and C# to implement htem in the exact same fashion.